tests/conftest.py

```diff
@@ -10,9 +10,122 @@
 
 
 EXTRA_DATA = b"\x124Vx\x124Vx\x124Vx\x124Vx"
-EXPIRATION_TIME = 4102444800  # 01/01/2100
+EXPIRATION_TIME = 4_102_444_800  # 01/01/2100
 
 
 @pytest.fixture(scope="session", autouse=True)
 def bind_contracts(contract_assets):
     tldeploy.core.contracts.data = contract_assets
+
+
+class CurrencyNetworkAdapter:
+    def __init__(self, contract):
+        self.contract = contract
+
+    def set_account(
+        self,
+        a_address,
+        b_address,
+        *,
+        creditline_given=0,
+        creditline_received=0,
+        interest_rate_given=0,
+        interest_rate_received=0,
+        is_frozen=False,
+        m_time=0,
+        balance=0,
+    ):
+        self.contract.functions.setAccount(
+            a_address,
+            b_address,
+            creditline_given,
+            creditline_received,
+            interest_rate_given,
+            interest_rate_received,
+            is_frozen,
+            m_time,
+            balance,
+        ).transact()
+
+    def check_account(
+        self,
+        a_address,
+        b_address,
+        *,
+        creditline_given=None,
+        creditline_received=None,
+        interest_rate_given=None,
+        interest_rate_received=None,
+        is_frozen=None,
+        m_time=None,
+        balance=None,
+    ):
+        account_tuple = self.contract.functions.getAccount(a_address, b_address).call()
+
+        for value, index in enumerate(
+            [
+                creditline_given,
+                creditline_received,
+                interest_rate_given,
+                interest_rate_received,
+                is_frozen,
+                m_time,
+                balance,
+            ]
+        ):
+            if value is not None:
+                if value != account_tuple[index]:
+                    return False
+            return True
```

Comment:
It's hard to see for me but I don't think this is properly indented

Reply:
yes, you are right. That happens when you dont use the code you write immediately. I'll make sure that this is used, and fix it.